src/crd/policies/admission_policy.rs

```diff
@@ -119,6 +119,11 @@ pub struct AdmissionPolicySpec {
     /// The timeout value must be between 1 and 30 seconds.
     /// Default to 10 seconds.
     pub timeout_seconds: Option<TimeoutSeconds>,
+
+    /// Message is  used to specify the message that will be returned when
+    /// the policy is rejected. The specific policy results will be
+    /// returned in the warning field of the response.
```

Comment:
This has to be updated to reflect what I've written on [this PR](https://github.com/kubewarden/kubewarden-controller/pull/1140#discussion_r2142607502)